src/functionalTest/resources/features/F-007/S-081.td.json

```diff
@@ -39,6 +39,50 @@
           "__ordering__": "UNORDERED",
           "__elementId__": "roleName"
         },
+        {
+          "id": "[[ANYTHING_PRESENT]]",
+          "actorIdType": "IDAM",
+          "actorId": "${[scenarioContext][testData][request][body][queryRequests][0][actorId][0]}",
+          "roleType": "ORGANISATION",
+          "roleName": "district-judge",
+          "classification": "PUBLIC",
+          "grantType": "STANDARD",
+          "roleCategory": "JUDICIAL",
+          "readOnly": false,
+          "beginTime": "[[ANYTHING_PRESENT]]",
+          "created": "[[ANYTHING_PRESENT]]",
+          "attributes": {
+            "substantive": "Y",
+            "contractType": "Salaried",
+            "primaryLocation": "[[ANYTHING_PRESENT]]",
+            "jurisdiction": "CIVIL",
+            "region": "[[ANYTHING_PRESENT]]",
+            "workTypes": "[[ANYTHING_PRESENT]]"
```

Comment:
should we specify expected workTypes in FTA rather than using `[[ANYTHING_PRESENT]]`?

Reply:
addressed in [1b799af](https://github.com/hmcts/am-org-role-mapping-service/pull/2147/commits/1b799af3c68e411a5eca17341995a224bf5fcc74)